lib/config/error-codes/throw.js

```diff
@@ -24,42 +24,87 @@ const { format } = require('util'),
   _ = require('lodash'),
   { errors } = require('kuzzle-common-objects');
 
-module.exports = {
-  getError: (...args) => {
+/**
+ * Construct and return the corresponding error
+ *
+ * @param  {string} domain - Domain (eg: 'external')
+ * @param  {string} subdomain - Subdomain (eg: 'elasticsearch')
+ * @param  {string} error - Error name: (eg: 'index_not_found')
+ * @param  {...any} placeholders - Placeholders value to inject in error message
+ */
+function getError (domain, subdomain, error, ...placeholders) {
```

Comment:
I wonder if we shouldn't simply name these functions: get, throw, reject...

Because we usually name this module `errorsManager`, so `errorsManager.getError` is kind of redundant.

Reply:
Yes I totally agree but I wouldn't do that in this PR because it will bloat all the diff